src/credentials/auth.ts

```diff
@@ -586,6 +587,48 @@ export class Auth implements AuthService, ConnectionManager {
         return this.#validationErrors.get(connection.id)
     }
 
+    /**
+     * Returns true if credential data can be authenticated successfully
+     */
+    public async canAuthenticateRawData(data: StaticProfile): Promise<boolean> {
```

Comment:
"raw data" as opposed to what?

Reply:
With the other available providers, the 'raw data' is encapsulated in something like env vars or the credentials file and the specific provider understands how to retrieve the data from them.

Let me know if you have a better name in mind